Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -240,6 +241,7 @@ void CLuaVehicleDefs::AddClass(lua_State* luaVM)
     lua_classfunction(luaVM, "getWheelScale", "getVehicleWheelScale");
     lua_classfunction(luaVM, "getModelWheelSize", "getVehicleModelWheelSize");
     lua_classfunction(luaVM, "getWheelFrictionState", "getVehicleWheelFrictionState");
+    lua_classfunction(luaVM, "getEntryPoints", ArgumentParser<GetVehicleEntryPoints>);
```

Comment:
It should be
```cpp
lua_classfunction(luaVM, "getEntryPoints", "getVehicleEntryPoints");
```

Reply:
ArgumentParser is required here

Reply:
Sure, but it's not possible in my use case as we cannot return vectors on non-OOP functions. Therefore I can't use the same function for both cases.

Reply:
> Sure, but it's not possible in my use case as we cannot return vectors on non-OOP functions. Therefore I can't use the same function for both cases.

You can return vectors. Being OOP doesnt matter. A good example of it is [`aclObjectGetGroups`](https://github.com/multitheftauto/mtasa-blue/blob/master/Server/mods/deathmatch/logic/luadefs/CLuaACLDefs.h) function

Reply:
> > Sure, but it's not possible in my use case as we cannot return vectors on non-OOP functions. Therefore I can't use the same function for both cases.
> 
> You can return vectors.

Different scenario - this is an array with vectors. I've ran more tests and the argument parser isn't mapping them correctly to an array of 3 floats that you'd expect from a non-OOP function. Modifying the argument parser to support this is out of the scope of this PR.

Reply:
> Different scenario - this is an array with vectors. I've ran more tests and the argument parser isn't mapping them correctly to an array of 3 floats that you'd expect from a non-OOP function. Modifying the argument parser to support this is out of the scope of this PR.

What about vector of vectors? (`std::vector<CVector>`)